templates/infinispan-ephemeral.json

```diff
@@ -200,29 +304,35 @@
                     "name": "hotrod",
                     "protocol": "TCP"
                   }
-                ]
+                ],
+                "volumeMounts": [
+                  {
+                    "name": "${APPLICATION_NAME}-configuration",
+                    "mountPath": "/opt/jboss/infinispan-server/standalone/configuration/custom"
+                  }
+                ],
+                "resources": {
+                  "requests": {
+                    "cpu": "0.5",
+                    "memory": "512Mi"
```

Comment:
No, no, those values are *minimum* CPU/Memory requirements for Infinispan to operate :)

Java based Data Grids are very resource hungry beasts. So it seems the best idea is to get into [Burstable QoS](https://docs.openshift.org/latest/dev_guide/compute_resources.html#quality-of-service-tiers) for both CPU (it affects replication speed and querying) and memory (it affects how much data you can store).